sbin/prepareWorkspace.sh

```diff
@@ -166,18 +169,36 @@ checkingAndDownloadingAlsa()
   fi
 }
 
-checkFingerPrint() {
+checkFingerprint() {
   local sigFile="$1"
   local fileName="$2"
   local publicKey="$3"
   local expectedFingerPrint="$4"
+  local expectedChecksum="$4"
```

Comment:
expectedChecksum == excpectedFingerPrint?  and FingerPrint should be Fingerprint